src/WebJobs.Script/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -24,15 +26,20 @@ public static bool IsHttpInAndOutFunction(this FunctionMetadata metadata)
 
             BindingMetadata inputBindingMetadata = metadata.InputBindings.ElementAt(0);
             BindingMetadata outputBindingMetadata = metadata.OutputBindings.ElementAt(0);
-            if (string.Equals("httptrigger", inputBindingMetadata.Type, StringComparison.OrdinalIgnoreCase) &&
-                string.Equals("http", outputBindingMetadata.Type, StringComparison.OrdinalIgnoreCase))
+            if (string.Equals(HttpTriggerKey, inputBindingMetadata.Type, StringComparison.OrdinalIgnoreCase) &&
```

Comment:
Adding the constants b/c this was bothering me, but since it does touch unrelated components (custom handler code), I can revert this section and keep the constants just for `IsHttpTriggerFunction`.